pom.xml

```
@@ -50,6 +54,12 @@
             <artifactId>junit-jupiter-engine</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>24.0.1</version>
```

Prefer to use a version-based property - makes it easier to keep versions up to date using the maven versions plugin.

```suggestion
            <version>${annotations.version}</version>
```

---

Agreed.